javascript/lib.c

```diff
@@ -82,6 +82,9 @@ napi_value api_cos(napi_env env, napi_callback_info info) {
 napi_value api_l2sq(napi_env env, napi_callback_info info) {
     return dense(env, info, simsimd_metric_sqeuclidean_k, simsimd_datatype_unknown_k);
 }
+napi_value api_l2eu(napi_env env, napi_callback_info info) {
```

Comment:
The convention is to call `l2` and `l2sq` or `euclidean` and `sqeuclidean`. The `l2eu` variant makes the codebase less searchable.

Reply:
ah, so should I just name it `api_l2` then ?

Reply:
Yes, let's do that.

Reply:
I have updated this to name it to just `api_l2`. It should be hopefully consistent now :)

Reply:
I have updated this to name it to just api_l2. It should be hopefully consistent now :)